storage/Utils/Logger.h

```diff
@@ -20,29 +20,28 @@
  * find current contact information at www.novell.com.
  */
 
-
 #ifndef STORAGE_LOGGER_H
 #define STORAGE_LOGGER_H
 
-
 #include <string>
 
-
 namespace storage
 {
-
-
     enum class LogLevel { DEBUG = 0, MILESTONE = 1, WARNING = 2, ERROR = 3 };
 
-
     /**
-     * The Logger class.
+     * Base Logger class
      */
     class Logger
     {
     public:
 
-	Logger() {}
+	static Logger* create(const std::string& filename = "");
```

Comment:
It is also unclear who owns the created instance; that should be documented here. If the caller owns it, is he supposed to delete it? Is the pointer stored in the class? Is it safe to be deleted? It the instance intentionally leaked? What happens if this `create()` is called multiple times? Is the old instance deleted in that case, or is the old instance leaked?

If this is only useful for the test suite, maybe it should rather be called someting like `create_testsuite_logger()`?